vendor/engines/projects/app/controllers/projects/projects_controller.rb

```diff
@@ -73,7 +74,13 @@ def edit
     end
 
     def show
-      @order_details = @project.order_details.order(ordered_at: :desc).paginate(page: params[:page])
+      @order_details = if @project.orders.any? && SettingsHelper.feature_on?(:cross_core_order_view)
+                         OrderDetail.joins(:order).where({ order: { cross_core_project_id: @project.id } })
```

Comment:
should we make this scope change on the Project models `order_details` association instead?